kythe/java/com/google/devtools/kythe/analyzers/java/JavaEntrySets.java

```diff
@@ -185,7 +185,7 @@ public EntrySet newClassInitAndEmit(VName classNode) {
         newNode(NodeKind.FUNCTION)
             .setCorpusPath(CorpusPath.fromVName(classNode))
             .addSignatureSalt(classNode)
-            .addSignatureSalt("cinit")
+            .addSignatureSalt("clinit")
```

Comment:
Yeah, I guess it doesn't matter. There'd be a mild aesthetic improvement if we didn't just have "salt" but also e.g. a "sigPrefix" on the builder, so the sig ended up looking something like like "#clinit-a93ef8g90a9"